docs/customization/model.rst

```diff
@@ -250,3 +250,167 @@ you'll need to update its form type. Check how to do it :doc:`here </customizati
+
+Just like for regular models you can also check the class of translatable models like that:
+
+.. code-block:: bash
+
+    $ php bin/console debug:container --parameter=sylius.model.shipping_method.class
+
+**1.** Appart from extending the ```ShippingMethod``` class as described before, you have to write your own class which will extend the base ``ShippingMethodTranslation`` class.
+
+.. code-block:: php
+
+    <?php
+
+    namespace AppBundle\Entity;
+
+    use Sylius\Component\Shipping\Model\ShippingMethodTranslation as BaseShippingMethodTranslation;
+
+    class ShippingMethodTranslation extends BaseShippingMethodTranslation
+    {
+        /**
+         * @var string
+         */
+        private $deliveryConditions;
+
+        /**
+         * @return string
+         */
+        public function getDeliveryConditions()
+        {
+            return $this->deliveryConditions;
+        }
+
+        /**
+         * @param string $deliveryConditions
+         */
+        public function setDeliveryConditions($deliveryConditions)
+        {
+            $this->deliveryConditions = $deliveryConditions;
+        }
+    }
+
+**2.** Next define your entity's mapping.
+
+The file should be placed in ``AppBundle/Resources/config/doctrine/ShippingMethodTranslation.orm.yml``
+
+.. code-block:: yaml
+
+    AppBundle\Entity\ShippingMethodTranslation:
+        type: entity
+        table: sylius_shipping_method_translation
+        fields:
+            deliveryConditions:
+                type: string
+                nullable: true
+
+**3.** You'll need to provide access to the new fields in the ```ShippingMethod``` class and initialize the translations.
+
+.. code-block:: php
+
+    <?php
+
+    namespace AppBundle\Entity;
+
+    //[...]
+    use Sylius\Component\Resource\Model\TranslatableTrait;
+
+
+    class ShippingMethod extends BaseShippingMethod
+    {
+        //[...]
+
+        use TranslatableTrait {
+            __construct as private initializeTranslationsCollection;
+        }
+        
+        public function __construct()
+        {
+            parent::__construct();
+            $this->initializeTranslationsCollection();
+        }
+
+        //[...]
+
+       /**
+         * Set delivery conditions
+         *
+         * @param string $deliveryConditions
+         */
+        public function setDeliveryConditions($deliveryConditions = null)
+        {
+            $this->getTranslation()->setDeliveryConditions($deliveryConditions);
+        }
+
+       /**
+         * Get delivery conditions
+         *
+         * @return string
+         */
+        public function getDeliveryConditions()
+        {
+            return $this->getTranslation()->getDeliveryConditions();
+        }
+
+       /**
+         * {@inheritdoc}
+         */
+        public static function getTranslationClass()
+        {
+            return ShippingMethodTranslation::class;
```

Comment:
But it is just for the information of the docs users. ;) If you think it is not needed (and it works) then leave it like you have it.